crates/astria-bridge-withdrawer/src/bridge_withdrawer/mod.rs

```diff
@@ -245,6 +248,47 @@ impl BridgeWithdrawer {
     }
 }
 
+type TaskHandles = (
+    JoinHandle<eyre::Result<()>>,
+    Option<JoinHandle<eyre::Result<()>>>,
+    JoinHandle<eyre::Result<()>>,
+    JoinHandle<eyre::Result<()>>,
+);
```

Comment:
Maybe cleaner to just create a struct, especially since most of the elements here are of the same type (harder that way to mix them up at the callsite of `spawn_tasks`).